lib/dai-plugin-mcd/src/CdpType.js

```diff
@@ -56,7 +56,13 @@ export default class CdpType {
   }
 
   async getLiquidationRatio() {
-    const { mat } = await this._getSpotInfo();
+    await this._getSpotInfo();
+    return this.getLiquidationRatioSync();
+  }
+
+  getLiquidationRatioSync() {
+    assert(this._spotInfo, '_spotInfo is not cached');
+    const { mat } = this._spotInfo;
```

Comment:
@jparklev so what i'm imagining is that in a front-end app using Multicall, these cache values could be populated by Multicall rather than `_getSpotInfo`. down the road we can have a proper API for that but in the short-term we could also just directly assign:
```js
someCdpType._spotInfo = spotInfoFromMulticall;
```

This would let us decouple data-fetching and logic in these classes in the plugin, so we can still use them regardless of how we're getting the data.